test/extended/cpu_partitioning/crio.go

```diff
@@ -120,6 +126,10 @@ var _ = g.Describe("[sig-node][apigroup:config.openshift.io] CPU Partitioning no
 		isClusterCPUPartitioned = getCpuPartitionedStatus(oc) == ocpv1.CPUPartitioningAllNodes
 	})
 
+	g.AfterEach(func() {
+		o.Expect(cleanup(managedNamespace, managedNamespace.Namespace())).NotTo(o.HaveOccurred())
```

Comment:
nit:
```
o.Expect(cleanup(managedNamespace, managedNamespace.Namespace())).tTo(o.Succeed())
```
Avoiding negatives when possible with inline expectations, a tad easier to process

Reply:
Oh fair point, it is easier to read, thanks! Updating